mtp_noms_ops/apps/security/urls.py

```diff
@@ -394,5 +403,5 @@ def dashboard_view(request):
             RedirectView.as_view(pattern_name='security:prisoner_list'),
         ),
         name='prisoner_list_legacy',
-    ),
+    )
```

Comment:
i disagree :) i think a trailing comma keeps blocks symmetrical so re-ordering and adding new ones makes the diff clearer (i.e. no lines are modified, you only get new lines or removed lines)

Reply:
I assure you this wasn't a conscious choice on my part ;) I prefer trailing commas everywhere too